_pkgdown.yml

```diff
@@ -1,10 +1,51 @@
 destination: docs
 title: worldfootballR
 url: https://jaseziv.github.io/worldfootballR/
+
+authors:
+  Jason Zivkovic:
+    href: https://twitter.com/jaseziv
+  Tony ElHabr:
+    href: https://twitter.com/TonyElHabr
```

Comment:
tbh i don't know how or where this is used in the site. The Twitter component is in the navbar, so maybe it's not all that important.